libs/components/popovers/src/lib/modules/dropdown/dropdown.component.html

```diff
@@ -8,12 +14,20 @@
     [attr.aria-label]="
       label ||
       (buttonType === 'context-menu'
-        ? (contentInfoObs | async)?.descriptor
+        ? (contentInfoObs | async)?.descriptor?.type === 'text'
           ? ('skyux_dropdown_context_menu_with_content_descriptor_default_label'
-            | skyLibResources : (contentInfoObs | async)?.descriptor)
+            | skyLibResources : (contentInfoObs | async)?.descriptor?.value)
           : ('skyux_dropdown_context_menu_default_label' | skyLibResources)
         : undefined)
     "
+    [attr.aria-labelledBy]="
```

Comment:
`aria-labelledby` should be all lower-case.

Reply:
huh, surprised that still worked

Reply:
Native HTML attributes aren't case-sensitive, but our convention is to use all lower-case for element and attribute names.